frontend/src/lib/k8s/apiProxy/apiUtils.ts

```diff
@@ -0,0 +1,67 @@
+import { omit } from 'lodash';
```

Comment:
Can we please move these into files named without utils?

Reply:
moved some functions around, only url formatting functions were left, renamed it to formatUrl.ts